### DISCUSSION TITLE
Using lazygit often comes up with a warning

### DISCUSSION THREAD

**Discussion by ghost** [jesseduffield/lazygit#3078] [Q&A]
When I open lazygit or pull, push code often shows a warning git output: Enter passphrase for key: git@github.com: Permission denied (publickey). fatal: Could not read from remote repository. Even though I filled in the correct password and pushed the operation to the repo successfully, it still shows this message. In addition, if I just hang lazygit there, it will still appear like that?

**Comment by mark2185** [comment]
Do you have `autoFetch` enabled?

Do you use an `ssh-agent` of some sort?
